web/src/beta/lib/reearth-ui/components/CodeInput/index.tsx

```diff
@@ -0,0 +1,70 @@
+import MonacoEditor, { OnMount } from "@monaco-editor/react";
+import { FC, useMemo, useRef, useCallback } from "react";
+
+import { fonts } from "@reearth/services/theme";
+
+export type CodeInputProps = {
+  value?: string;
+  language?: string;
+  showLines?: boolean;
+  disabled?: boolean;
+  onChange?: (val: string | undefined) => void;
+  onBlur?: (val: string | undefined) => void;
+};
+
+export const CodeInput: FC<CodeInputProps> = ({
+  value = "{}",
```

Comment:
```suggestion
  value,
```
Maybe remove this default value since `{}` doesn't make sence when it comes to some other languages.